shared/attributes.asciidoc

```diff
@@ -87,6 +81,21 @@
 :graph-forum:          https://discuss.elastic.co/c/x-pack/graph
 :apm-forum:            https://discuss.elastic.co/c/apm
 
+//////////
+Elastic Cloud
+//////////
+:ecloud:      Elastic Cloud
+:ess:         Elasticsearch Service
+:ece:         Elastic Cloud Enterprise
+:cloud:       https://www.elastic.co/guide/en/cloud/current
+:ess-trial:   https://www.elastic.co/cloud/elasticsearch-service/signup?baymax=docs-body&elektra=docs
+:ess-product: https://www.elastic.co/cloud/elasticsearch-service?baymax=docs-body&elektra=docs
+:ess-console: https://cloud.elastic.co?baymax=docs-body&elektra=docs
+:ess-baymax:  ?baymax=docs-body&elektra=docs
+:ece-ref:     https://www.elastic.co/guide/en/cloud-enterprise/current
+:ess-leadin: You can run Elasticsearch on your own hardware or use our hosted Elasticsearch Service that is available on AWS, GCP, and Azure. {ess-trial}[Try the Elasticsearch Service for free].
```

Comment:
`:ess-leadin:` is the piece I need for Logstash.  It works like a champ! 
I've opened two Logstash PRs to make use of this attribute when this work gets merged.  Thanks for your work on this, @nrichers.  <3

Reply:
Glad it works, @karenzone, and let me know if you want additional text snippets!